coderd/audit.go

```diff
@@ -154,6 +154,9 @@ func (api *API) generateFakeAuditLog(rw http.ResponseWriter, r *http.Request) {
 		Diff:             diff,
 		StatusCode:       http.StatusOK,
 		AdditionalFields: params.AdditionalFields,
+		OrganizationID:   uuid.Nil,
```

Comment:
Oh, shouldn't we insert OrganizationID? I'm thinking about this: https://github.com/coder/coder/commit/9e845213f59f2eae704d15e1c69a0ad9a5dbe4b6

Reply:
Good point, makes sense to at least insert something fake here.